src/main/java/net/smartcosmos/cluster/auth/AuthServerApplication.java

```diff
@@ -186,57 +199,62 @@ private Filter csrfHeaderFilter() {
         private SecurityResourceProperties securityResourceProperties;
 
         @Autowired
-        private AuthenticationManager authenticationManager;
-
-        @Autowired
         private SmartCosmosAuthenticationProvider smartCosmosAuthenticationProvider;
 
         @Bean
         public JwtAccessTokenConverter jwtAccessTokenConverter() {
 
-            Assert.hasText(securityResourceProperties.getKeystore().getKeypair());
-            Assert.notNull(securityResourceProperties.getKeystore().getLocation());
+            Assert.hasText(securityResourceProperties.getKeystore()
+                               .getKeypair());
+            Assert.notNull(securityResourceProperties.getKeystore()
+                               .getLocation());
 
             JwtAccessTokenConverter converter = new JwtAccessTokenConverter();
             SmartCosmosUserAuthenticationConverter smartCosmosUserAuthenticationConverter = new SmartCosmosUserAuthenticationConverter();
 
-            smartCosmosUserAuthenticationConverter
```

Comment:
https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework/pull/216 actually depended on this change here.